packages/interaction-kit/src/interactions/index.ts

```diff
@@ -139,15 +123,15 @@ async function handleMessageComponentInteraction(
 		);
 
 		console.log(`Handling ${interaction.customId}`);
-		return component.handler(interaction, application);
+		component.handler(interaction, application);
+	} else {
+		throw new Error(
+			`Unknown Interaction Component type (${
+				// eslint-disable-next-line @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-member-access
```

Comment:
fyi the fact that ts doesn't complain here suggests something is missed, probably deal with it in another PR though